src/common/context/forms/index.tsx

```diff
@@ -57,9 +60,30 @@ export const FormsContextProvider: FunctionComponent = ({ children }) => {
         ownership: { beneficiaryAddress: "", holderAddress: "" },
       },
     ]);
+
     setActiveFormIndex(newIndex);
   };
 
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
i see liberal usage of eslint rule disabling in this PR, are they all necessary?

Reply:
This is to prevent the eslint to show warning/error due to the function's variable argument is set to type "any"

Reply:
yes i know what the rule is for :stuck_out_tongue: 
lint rules generally exist to keep you from doing bad things so you shouldn't be using it unless you have a very good reason

should you be using 'any' in all the cases here?

Reply:
Okay, I will go check on the exact variable type and check if there are any issues if it is implemented that way.

Reply:
Done.